action/steps/build.sh

```diff
@@ -111,6 +111,7 @@ echo ""
 xvfb-run --auto-servernum --server-args='-screen 0 640x480x24' \
   /opt/Unity/Editor/Unity \
     -batchmode \
+    -nographics \
```

Comment:
don't forget about the other scripts that also run unity

Reply:
this flag is already there at other places (4 places in total) just here it wasn't